lib/import/rummager_data_presenter.rb

```diff
@@ -7,6 +7,7 @@ def self.present_content(batch_data)
           item['content_id'],
           item['format'],
           item['index'],
+          item['is_withdrawn'] == 'true' ? 'withdrawn' : 'not_withdrawn',
```

Comment:
The string is coming from a result object from gds-api-adapters.
I didn't find docs to replace the types I would ordinarily look at to know what I have in my response, so we checked in the console. Please correct us if you think maybe we looked at the wrong thing!

I agree that raw strings are a very bad habit. What is the idiomatic way of packaging constant values in ruby? A module of symbol constants maybe?